src/main/java/org/isf/dicomtype/gui/DicomTypeEdit.java

```diff
@@ -302,8 +302,7 @@ private JPanel getJDataPanel() {
 	 */
 	private JLabel getJCodeLabel() {
 		if (jCodeLabel == null) {
-			jCodeLabel = new JLabel();
-			jCodeLabel.setText(MessageBundle.getMessage("angal.dicomtype.codemaxchars"));
+			jCodeLabel = new JLabel(MessageBundle.getMessage("angal.dicomtype.codemaxchars"));
```

Comment:
how many chars? no generalization here?

Reply:
Fixed (although GUI changes were not the primary focus of this PR).